lxd/util/kernel.go

```diff
@@ -10,13 +10,13 @@ import (
 )
 
 // LoadModule loads the kernel module with the given name, by invoking
-// modprobe.
+// modprobe. This respects any modprobe configuration on the system.
 func LoadModule(module string) error {
 	if shared.PathExists(fmt.Sprintf("/sys/module/%s", module)) {
 		return nil
 	}
 
-	_, err := shared.RunCommand("modprobe", module)
+	_, err := shared.RunCommand("modprobe", "-b", module)
```

Comment:
Does this error if the module is blacklisted or should we check if its loaded afterwards?

Reply:
It may error depending on the configuration.

We can't actually check afterwards as modprobe may give us a success to tell us the module is aliased or built-in, neither of those would result in a successful filesystem test despite in theory providing the kernel functionality we asked for.